kea-vscode/src/views/pull-request-contents/commits/local-commit-tree-node.ts

```diff
@@ -0,0 +1,164 @@
+import * as vscode from "vscode";
+import { ILocalGitRepository, LocalCommit } from "../../../git/local-git-repository";
+import { CollapsibleState, getCollapsibleState, IParentTreeNode } from "../../tree-node";
+
+// Create a simplified local file tree node for local commits
+export class LocalFileTreeNode {
```

Comment:
Please fix the following:

- Please only show the commits which are part of the branch, and are ahead of the target branch.
  - The files for the Files node under the top-level pull request have the same issue
- The files for the left-hand side of diffs for the last commit are the same as the right-hand side. This suggests that the old files are not being retrieved correctly.
- The files on the left-hand side don't have syntax highlighting.
- Separate out the tree nodes for the local files and local folders into separate folders.
- Add full stops to all comments.
- Make sure that TypeScript type-checking passes.